### PR TITLE
FDRS: Fetch only the latest year related data

### PR DESCRIPTION
**Pull request by susilnem**
Addresses
- https://github.com/IFRCGo/go-web-app/issues/1396
- https://github.com/IFRCGo/go-api/issues/2285

## Changes
-  `KPI_noLocalUnits` changed to `KPI_noBranches`
- Updated the cron job databank -> FDRS
This pull request's related to `databank/management/commands/sources/FDRS.py`.
- Change the field name in fdrs and worldbank related fields.
- Add new migrations file.

## Need to run these commands.
`python manage.py ingest_databank` and `python manage.py ingest_worldbank`

Improvements to data fetching and processing:

- Changed the key from `KPI_noLocalUnits` to `KPI_noBranches`.

- Updated the `prefetch` function to fetch the latest data for each indicator by using the `max` function to get the entry with the highest year. (`databank/management/commands/sources/FDRS.py`)
-  Modified the `load` function to ensure only the latest year's available data is used by checking the year before setting the value. (`databank/management/commands/sources/FDRS.py`) 

## Checklist
Things that should succeed before merging.

- [x] Updated/ran unit tests
- [x] Updated CHANGELOG.md

## Release

If there is a version update, make sure to tag the repository with the latest version.

### CODE REVIEW

**Review by szabozoltan69 — APPROVED** [review]

File: databank/management/commands/sources/FDRS.py
Comment (thenav56) [inline databank/management/commands/sources/FDRS.py:136]: We will also need to clear the field's value if it doesn't exist in the remote source for the max year.

File: databank/management/commands/sources/FDRS.py
Comment (thenav56) [inline databank/management/commands/sources/FDRS.py:28]: > Issue comment: https://github.com/IFRCGo/go-web-app/issues/1396#issuecomment-2388892005
> So, it has been just updated (KPI_noBranches) in the Data2 FDRS API (note that we only started collecting this information for 2023)

This needs to fetch from different endpoint?

Maybe this? https://data-api.ifrc.org/api/data2?apiKey=XX-YY-ZZ&KPI_Don_Code=DAF001&indicator=KPI_noBranches

**Review by thenav56 — CHANGES_REQUESTED** [review]

File: databank/management/commands/sources/FDRS.py
Comment (thenav56) [reply databank/management/commands/sources/FDRS.py:28]: This is already fixed.

**Review by thenav56 — APPROVED** [review]
Let's wait until approval from QA

**Review by Rup-Narayan-Rajbanshi — APPROVED** [review]

**Review by szabozoltan69 — APPROVED** [review]

**Comment by szabozoltan69** [comment]
Should we merge this?

**Comment by susilnem** [comment]
@szabozoltan69 
we also need to run these commands:
`python manage.py ingest_databank` and `python manage.py ingest_worldbank`

**Comment by szabozoltan69** [comment]
Both were run fine (ingest_databank had a slight issue).

**Pull request merged**
